src/FsAutoComplete.Core/Commands.fs

```diff
@@ -985,20 +987,11 @@ type Commands<'analyzer> (serialize : Serializer, backgroundServiceEnabled) =
             // ENHANCEMENT: consider caching the Fantomas configuration and reevaluate when the configuration file changes.
             let config =
                 let currentFolder = Path.GetDirectoryName(file)
-                let result = Fantomas.CodeFormatter.ReadConfiguration currentFolder
-                match result with
-                | Fantomas.FormatConfig.Success c -> c
-                | Fantomas.FormatConfig.PartialSuccess(c,warnings) ->
-                    match warnings with
-                    | [] ->
-                      c
-                    | warnings ->
-                      fantomasLogger.warn (Log.setMessage "Warnings while parsing the configuration file at {path}" >> Log.addContextDestructured "path" currentFolder >> Log.addContextDestructured "warnings" warnings)
-                      c
-                | Fantomas.FormatConfig.Failure err ->
-                    fantomasLogger.error (Log.setMessage "Error while parsing the configuration files at {path}. Using default configuration" >> Log.addContextDestructured "path" currentFolder >> Log.addExn err)
-                    Fantomas.FormatConfig.FormatConfig.Default
-
+                match Fantomas.CodeFormatter.TryReadConfiguration currentFolder with
```

Comment:
@nojaf could you sanity-check this change? I believe it's correct (and yes I know about ReadConfiguration instead, but I wanted to log when there's not one found).

Quick personal question: does this look in the current directory and parent directories as editorconfigs expect?

Reply:
You should pass the F# file path instead of the folder here.
It will then search for settings within .editorconfig files. In theory, you might have different settings for certain extensions. It will also check parent folders (until one .editorconfig is marked as `root=true`).

I think it is ok here to use `CodeFormatter.ReadConfiguration` this will take the default settings if nothing was found in .editorconfig search. Unless you really want to log that nothing was found. 
We don't warn anymore for unknown settings (as .editorconfig) can have many unrelated settings.